platform/ios/src/MGLMapView.h

```diff
@@ -133,6 +133,25 @@ FOUNDATION_EXTERN MGL_EXPORT const MGLMapViewPreferredFramesPerSecond MGLMapView
 FOUNDATION_EXTERN MGL_EXPORT MGLExceptionName const MGLMissingLocationServicesUsageDescriptionException;
 FOUNDATION_EXTERN MGL_EXPORT MGLExceptionName const MGLUserLocationAnnotationTypeException;
 
+/**
+ A structure containing information about the default User Location view style.
+ */
+typedef struct __attribute__((objc_boxable)) MGLUserLocationStyle {
```

Comment:
I would encourage documenting each of these properties so that the developer has a good sense of what parts of the user location annotation are changing - some things like the `approximate` properties are new and perhaps not widely understood yet.

Reply:
I added the documentation.